tests/test_dimacs.py

```diff
@@ -0,0 +1,37 @@
+from mis.data.graphs import load_dimacs
+
+
+def test_hexagon():
```

Comment:
I guess you can also check that the solution found is genuinely and independent set by using [this method](https://github.com/pasqal-io/maximum-independent-set/blob/08b0b9ff6875acd7dc344c60e2be5225cc8ed4fa/mis/pipeline/kernelization.py#L58).

Reply:
Ok done